custom_components/tuya_local/devices/bonaire_heater.yaml

```diff
@@ -0,0 +1,119 @@
+name: Bonaire Electric Heater
```

Comment:
Leave the branding out of this top level name. And maybe reduce the duplication in the name below under products, so that one only includes brand and model info (no need to repeat what the above says).